### PR TITLE
changed the card on the event index

### PR DESCRIPTION
**Pull request by royyirao2017**

### CODE REVIEW

File: pages/user/user.wxml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!--index.wxml-->
 <view class="container">
-  <view  bindtap="bindViewTap" class="userinfo">
+
+
+
+df  <view  bindtap="bindViewTap" class="userinfo">
```
Comment (forresty) [inline pages/user/user.wxml]: what is `df` here?

File: pages/user/user.wxml [hunk 2]
```diff
@@ -14,16 +17,16 @@
   <view class="section__events">
 
     <block wx:for="{{events}}" wx:for-item="event">
-      <navigator url="/pages/event/event?id={{ event.id }}">
+      <navigator url="/pages/event/event">
```
Comment (forresty) [inline pages/user/user.wxml]: obviously this line breaks existing feature. please always review your own PR first. @royyirao2017

File: pages/user/user.wxml [hunk 3]
```diff
@@ -14,16 +17,16 @@
...
         <view class="flex-wrp section__events__event" style="flex-direction:row;">
           <view class="flex-item event-poster">
-            <image src="{{event.poster}}"></image>
+            <image src="http://img02.tooopen.com/images/20150928/tooopen_sy_143912755726.jpg"></image>
```
Comment (forresty) [inline pages/user/user.wxml]: also this line

**Review by forresty — CHANGES_REQUESTED** [review]
see inline messages

File: pages/user/user.wxml (hunk 1 above)
Comment (royyirao2017) [reply pages/user/user.wxml]: no need

**Review by forresty — APPROVED** [review]
LGTM

**Review by dengel29 — APPROVED** [review]

**Pull request merged**